skopt/optimizer/optimizer.py

```diff
@@ -528,3 +460,95 @@ def run(self, func, n_iter=1):
 
         return create_result(self.Xi, self.yi, self.space, self.rng,
                              models=self.models)
+
+    def fit(self, mask=None):
+        """Fit a model to observed evaluations of the objective.
+
+        Parameters
+        ----------
+        * `mask` [None or list]
```

Comment:
For me `mask` and arrays suggests masking out elements in an array, not really fixing them to a certain value. I will try and make a suggestion for a name I think would be a better fit. If anyone else has ideas post them :)

Reply:
That's a good point. Yes, mask is perhaps not the best name.